### ISSUE TITLE
Implement CNN to collect logs from wood

### ISSUE THREAD

**Issue by andrefdre** [andrefdre/AI-Enhanced-Minecraft#12]
The simplest task we can try and where we should begin is by implementing a cnn to capture logs. To do this will have game image as input and keys as output.

**Comment by andrefdre** [comment]
Currently, when I tried to train the model I get an error saying `0D or 1D target tensor expected, multi-target not supported` which I think is due to the current loss function doesn't support this type of structure. Meaning that we can't pass an array with keystrokes.

I think we need to solve this by research existing loss functions or think about developing one.

**Comment by andrefdre** [comment]
For the true or false, we can use [binary cross-entropy](https://towardsdatascience.com/cross-entropy-for-classification-d98e7f974451) that allows to give multiple labels.
